app.ts

```diff
@@ -223,6 +228,7 @@ async function attachToReport(reportId: string, fileUri: string) {
     ?report prov:value ?document .
   } INSERT {
     ?report prov:value ${sparqlEscapeUri(fileUri)} .
+    ?report dct:modified ${sparqlEscapeDate(new Date())}
```

Comment:
**chore**: use `sparqlEscapeDateTime`

`sparqlEscapeDate` returns a (formatted) date, without a timestamp. We want the timestamp too, otherwise modifications happen at the granularity of a day.